### ISSUE TITLE
Feature Request : New command - zectl chroot

### ISSUE THREAD

**Issue by GregoryLand** [johnramsden/zectl#31]
I would like to request a new feature that I think might be useful.  When working with some machines I have zectl on I have setup a "recovery" rootfs.  This exists on all of my systems and I use it bring up the system.  Afterwards I clone this environment and use the clone to build out the rest of the system.  If the system ever ends up in an un-bootable state I can use this as a starting point to fix issues.  In the middle of this recovery process and maybe other processes I think it might be useful to be able to chroot into another "env".  It would be great if zectl could handle the process of setting up the mounts and everything necessary to allow me to chroot into another root and run things like pacman to correct errors in the system.  I have not put much thought into how something like this might exactly work though.

**Comment by johnramsden** [comment]
I agree this would be a nice feature, I will put it on my list of potential improvements.

One concern I have is making sure it works correctly on multiple different distros.

**Comment by duskmoss** [comment]
Something that might enhance its functionality - and might even make it more portable between distros:

instead of a standard chroot boot an lxc with it as the fs?
